src/tools/toolfactory.cpp

```diff
@@ -92,9 +92,12 @@ CaptureTool* ToolFactory::CreateTool(CaptureToolButton::ButtonType t,
         case CaptureToolButton::TYPE_REDO:
             tool = new RedoTool(parent);
             break;
+#if not(defined(Q_OS_MAC) || defined(Q_OS_MAC64) || defined(Q_OS_MACOS) ||     \
```

Comment:
No ability in Mac to open in external application?

Reply:
In our case - there's no need :)